assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -117,7 +123,7 @@ const CartLineItemRow = ( { lineItem } ) => {
 				<div className="wc-block-cart-item__product-name">{ name }</div>
 				{ lowStockBadge }
 				<div className="wc-block-cart-item__product-metadata">
-					{ description }
+					<div dangerouslySetInnerHTML={ { __html: summary } } />
```

Comment:
You can use `RawHTML` from `@wordpress/element` instead here.

```suggestion
					<RawHTML>{ summary }</RawHTML>
```